src/main/java/org/folio/oaipmh/helpers/AbstractHelper.java

```diff
@@ -315,6 +354,25 @@ private String getUntilDate(Request request, String from) {
     }
   }
 
+  protected boolean filterInstance(Request request, JsonObject instance){
```

Comment:
Could you please add javadoc and describe what happens here. It would be useful.